libfreerdp/core/settings.c

```diff
@@ -559,6 +559,12 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 	if (!settings->DynamicChannelArray)
 		goto out_fail;
 
+	settings->TcpKeepAlive = TRUE;
+	settings->TcpKeepAliveRetries = 3;
+	settings->TcpKeepAliveDelay = 2;
```

Comment:
Delay used to have value 5 in original code and Interval value 2 - think these two got mixed up in the PR